proto/panacea/oracle/v2/tx.proto

```diff
@@ -48,12 +48,12 @@ message MsgRegisterOracleResponse {
 
 // MsgApproveOracleRegistration defines the Msg/ApproveOracleRegistration
 message MsgApproveOracleRegistration {
-  ApproveOracleRegistration approve_oracle_registration = 1;
+  ApprovalSharingOracleKey approval_sharing_oracle_key = 1;
```

Comment:
How about renaming like `SharingOracleKeyApproval`?

Reply:
Or.. `MsgEncryptedOraclePrivKey`..?

Reply:
Hmm.... I think that the `ApprovalSharingOracleKey` is more better. 😂